test/big_query_helper.rb

```diff
@@ -3,3 +3,32 @@ def get_bq_row_count(study)
   bq_dataset = ApplicationController.big_query_client.dataset(CellMetadatum::BIGQUERY_DATASET)
   bq_dataset.query("SELECT COUNT(*) count FROM #{CellMetadatum::BIGQUERY_TABLE} WHERE study_accession = '#{study.accession}'", cache: false)[0][:count]
 end
+
+# seed BQ with 5 rows of data for a given study/metadata file
+def seed_bq(study, metadata_file)
```

Comment:
I don't think you need the 'metadata_file' argument, since you could get that from just calling study.metadata_file.  
Also, consider 'seed_example_bq_data' as a name that indicates we're not parsing the metadata file